great_expectations/expectations/expectation.py

```diff
@@ -284,6 +284,7 @@ class Config:
     meta: Union[dict, None] = None
     notes: Union[str, List[str], None] = None
     result_format: Union[ResultFormat, dict] = ResultFormat.BASIC
+    description: ClassVar[Union[str, None]] = None
```

Comment:
Explicitly making a `ClassVar` because I want it to be defined within the custom class (not overridden during instantiation)